### PR TITLE
added route state variable

### PR DESCRIPTION
**Pull request by ar5had**
https://github.com/Greennav/GreenNav/issues/32

### CODE REVIEW

**Comment by arshdkhn1** [comment]
@fabianbormann I want to understand the desired behavior of the `GET ROUTES` button. 
``` js
// this.state.route.every(waypoint => waypoint !== '') evaluates to true even if route is an empty array
if (this.state.route.every(waypoint => waypoint !== '')) {
      this.props.getRoute(this.state.route);
} else {
...
}
```

I guess the the else case should be executed if the route array is empty ?

**Comment by arshdkhn1** [comment]
``` js
const route = this.state.route;
if (route.length && route.every(waypoint => waypoint !== '')) {
     this.props.getRoute(this.state.route);
}
```

File: src/GreenNav.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import Dialog from 'material-ui/Dialog';
 import Toggle from 'material-ui/Toggle';
 import SelectField from 'material-ui/SelectField';
 import MenuItem from 'material-ui/MenuItem';
+import fetch from 'unfetch';
```
Comment (ar5had) [inline src/GreenNav.js:10]: Polyfill that supports fetch in older browsers!

**Comment by ar5had** [comment]
@fabianbormann I need to ask some questions please arrange some kinda session or chat so that all the doubts can be cleared. Thanks.

**Pull request merged**